inc/define.php

```diff
@@ -463,3 +463,7 @@
 //Maximum time, in miliseconds a saved search should not exeed
 //so we count it on display (using automatic mode).
 $CFG_GLPI['max_time_for_count'] = 200;
+
+if (!defined('GLPI_CACHE_CONFIG')) {
+   define('GLPI_CACHE_CONFIG', ['adapter' => 'apcu']);
```

Comment:
IIUC, the value can be defined in "local_define.php" ?

I haven't even notice this new file, but this seems messy.
Configuration is usually stored in the DB.
The only files in CONFIG_DIR are config_db (and slave) by design, as not storable in the DB

If sometime we add a UI to select the cache adapter (with connexion param for such case as redis, memcached...) how do the UI will save it ?

A GLPI_CONFIG_DIR/glpi_config_cache.php could work... but DB seems better (excepted if you can imagine some case where various PHP backends use the same DB but with different env, e.g. one server with APCu, one without)

Reply:
This is a very advanced use case, and aslo more a system configuration than a user one.

Also, writing a configuration form for that would be a too huge job for the goal (I had to work on this, using some conf was just an extra that may helps).

If you think we should really provide UI to configure that (I do not tink), I'll open another ticket; and this will not be done for 9.2.

Reply:
All system configurations are stored in the DB and managed from the UI (DB master during install, DB replica, ldap, mail server, mail receiver, ...)

Of course the UI can be a feature of next major version